stable/jenkins/values.yaml

```diff
@@ -6,7 +6,7 @@
 Master:
   Name: jenkins-master
   Image: "gcr.io/kubernetes-charts-ci/jenkins-master-k8s"
-  ImageTag: "v0.6.0"
+  ImageTag: "v0.7.0"
```

Comment:
@seanknox Agreed, they are valueless without updating the image. I have spoken with @lachie83 on Slack about the build process, and while I realize automating that with dependable security may be a longer-term effort, the fact that it has taken so long to do a simple image update means perhaps there should be more than one person with ability to do this. I would be happy to do so (as have done in my test image mentioned in [this comment](https://github.com/kubernetes/charts/pull/733#issuecomment-287483737). Is there someone else already on the team who can do this in the meantime, until the build process is reworked?